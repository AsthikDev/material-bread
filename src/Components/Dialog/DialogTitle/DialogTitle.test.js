import React from 'react';
import { DialogTitle } from '../../../';

import renderer from 'react-test-renderer';

test('DialogTitle Renders', () => {
  const tree = renderer.create(<DialogTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
