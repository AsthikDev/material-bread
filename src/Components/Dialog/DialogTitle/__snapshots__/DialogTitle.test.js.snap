// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogTitle Renders 1`] = `
<Text
  style={
    Object {
      "color": "rgba(0, 0, 0, 0.87)",
      "fontSize": 20,
      "fontWeight": "500",
      "marginBottom": 20,
      "paddingHorizontal": 24,
    }
  }
/>
`;
